src/keycloak/urls_patterns.py

```diff
@@ -155,6 +155,8 @@
     + "exportGroupsAndRoles={export-groups-and-roles}"
 )
 
+URL_ADMIN_REALM_PARTIAL_IMPORT = "admin/realms/{realm-name}/partialImport"
```

Comment:
Could you utilize `URL_ADMIN_REALM`? i.e. `URL_ADMIN_REALM_PARTIAL_IMPORT = URL_ADMIN_REALM + "/partialImport"`:)